cmd/prober/prober.go

```diff
@@ -277,6 +314,10 @@ func runProbers(ctx context.Context, freq int, runOnce bool, fulcioGrpcClient fu
 				hasErr = true
 				Logger.Errorf("error running rekor write prober: %v", err)
 			}
+			if err := rekorV2WriteEndpoint(ctx, cert, priv); err != nil {
```

Comment:
TODO: loop through known active writable shards and write to each of them: add rekorV2URL as an argument.

Reply:
fixed.